deploy/tests/test_utils.py

```diff
@@ -0,0 +1,69 @@
+from pathlib import Path
+from unittest.mock import MagicMock, call
+from botocore.exceptions import ClientError
+import pytest
+import os
```

Comment:
```suggestion
```

I think the only uses of `os` were misguided uses of stuff that are better done with the `Path` methods.